ci/jenkins/test-vmc.sh

```diff
@@ -301,6 +315,10 @@ function deliver_antrea {
         make manifest-coverage -C $GIT_CHECKOUT_DIR
         antrea_yml="antrea-coverage.yml"
     fi
+    if [[ -n $OLD_ANTREA_VERSION ]]; then
+        # Let antrea controller use old Antrea image
+        sed -i "0,/antrea-ubuntu:latest/{s/antrea-ubuntu:latest/antrea-ubuntu:$OLD_ANTREA_VERSION/}" ${GIT_CHECKOUT_DIR}/build/yamls/$antrea_yml
+    fi
```

Comment:
why not use the release manifests, which reference the correct docker image tag?

for example: `https://github.com/vmware-tanzu/antrea/releases/download/${OLD_ANTREA_VERSION}/antrea.yml`

Reply:
Because agents need to use the image with latest tag rather than the OLD_ANTREA_VERSION image.